great_expectations/compatibility/sqlalchemy.py

```diff
@@ -12,6 +12,11 @@
 except ImportError:
     sqlalchemy = SQLALCHEMY_NOT_IMPORTED
 
+try:
+    from sqlalchemy.sql.selectable import Subquery
+except ImportError:
```

Comment:
@SaeedFarahani We also need to add `AttributeError` here -- please see other examples that use it in this module.  Thanks!

Reply:
I fixed this as well